builtin/logical/pki/path_tidy.go

```diff
@@ -1459,6 +1496,20 @@ func (b *backend) pathConfigAutoTidyWrite(ctx context.Context, req *logical.Requ
 		}
 	}
 
+	if revokedSafetyBufferRaw, ok := d.GetOk("revoked_safety_buffer"); ok {
+		revokedSafetyBuffer := time.Duration(revokedSafetyBufferRaw.(int)) * time.Second
+		config.RevokedSafetyBuffer = &revokedSafetyBuffer
+		if *config.RevokedSafetyBuffer < 1*time.Second {
+			return logical.ErrorResponse(fmt.Sprintf("revoked_safety_buffer must be at least one second; got: %v", revokedSafetyBufferRaw)), nil
+		}
+	} else {
+		// If no explicit value for revoked_safety_buffer, default it to SafetyBuffer
+		if config.RevokedSafetyBuffer == nil {
```

Comment:
Hmm, I think this conditional is wrong. Consider:

```
$ bao read pki/config/auto-tidy
Key                                         Value
---                                         -----
... snip ...
revoked_safety_buffer                       259200
safety_buffer                               259200
... snip ...
$ bao write pki/config/auto-tidy safety_buffer=5s
Key                                         Value
---                                         -----
revoked_safety_buffer                       5
safety_buffer                               5
$ bao write pki/config/auto-tidy safety_buffer=8s
Key                                         Value
---                                         -----
revoked_safety_buffer                       5
safety_buffer                               8
```

In particular, the value for `revoked_safety_buffer` gets stuck on an old value of `safety_buffer` because of this `nil` check. But TBH, I'm not sure how to fix that. :-D

I _think_ ideally we'd leave the value nil until explicitly set?

That would require us to update logic in a lot of places, e.g., to set `revoked_safety_buffer = safety_buffer` if it is `nil` on display... but never in the actual config itself. Which I think is technically the most correct but also a fair bit of work. 

What's your thoughts? I'm happy to do that in a follow-up PR myself if you don't want to deal with that :D

Reply:
I wrote a quick test for this scenario (and pushed it), and it initially failed. After reading your suggestion to handle `revoked_safety_buffer` as display-only, I removed the assignment `revokedSafetyBuffer = safetyBuffer` from the getter in **storage.go** and handled it in **path_tidy.go** where needed. The tests are passing now :) 

I tried out a few other scenarios on my own (like only setting revoked and not safety) and they passed, I hope I haven't missed out on any other use cases... if this solution is still lacking, idm if you open up a follow up PR for it or leave suggestions for me :D Thank you!